code-samples/eventing/bookstore-sample-app/solution/solution.sh

```diff
@@ -102,13 +102,19 @@ kubectl apply -f config
 echo ""
 echo "⏳ Waiting for the frontend to be ready..."
 kubectl wait --for=condition=ready pod -l app=bookstore-frontend --timeout=300s
+kubectl wait --for=jsonpath='{.status.loadBalancer.ingress}' svc/bookstore-frontend-svc --timeout=300s
 echo "✅ Bookstore Frontend installed."
```

Comment:
Adding a check to ask if the user have configured the [KinD Loadbalancer](https://github.com/knative/docs/pull/6444/files#diff-2a5c3c0446ff1da8de7f92e02f696d0aef8bb1ebe7dff0b14df99e3960a28f48R128-R132) or minikube tunnel.

When I was running through the tutorial, I forgot to setup the minikube tunnel, and the script stuck there waiting to get the LoadBalancerIP.

Reply:
I added a log line to remind folks